### PR TITLE
Automate generating standards page

### PR DESCRIPTION
**Pull request by miklasr**
# Details

- Makes generating the page for the NMW standard automatic (easier to add new entries)
- Added check in CI to make sure docs version of standard matches app version
  - (Limitation of Pages means this need to be duplicated)
- Additions to the standard
- Added version label to NMW Standard page
- Allow using all icon packs
  - Specify icon field in nmwstandard.json as `"icon": "pack/icon"`

# Checklist

## Documentation
- [ ] I have updated documentation appropriately (GitHub Pages)
- [x] I have tested the updated GitHub Pages locally to ensure the changes to documentation render correctly (see this [Guidance](https://ese-peasy.github.io/PosturePerfection/guidance.html) page)

### CODE REVIEW

File: app/nmwstandard.json [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+    {"code": "1-4-1", "description": "Stairs: keep left", "icon": "stairs"},
+    {"code": "1-4-2", "description": "Stairs: keep right", "icon": "stairs"},
+    {"code": "1-5-0", "description": "Elevator", "icon": "elevator"},
+    {"code": "1-5-1", "description": "Elevator", "icon": "elevator"},
+    {"code": "1-5-2", "description": "Wheelchair elevator", "icon": "elevator"},
+    {"code": "1-5-3", "description": "Chairlift", "icon": "elevator"},
+    {"code": "1-6-0", "description": "Public toilet", "icon": ""},
+    {"code": "1-6-1", "description": "Public toilet", "icon": ""},
+    {"code": "1-6-2", "description": "Public toilet (single occupancy)", "icon": ""},
+    {"code": "1-6-3", "description": "Public washroom with shower", "icon": ""},
```
Comment (Ashwin-MJ) [inline app/nmwstandard.json]: I think we need to have icons defined or the app will not display anything in the area we have allocated for the icon

File: app/nmwstandard.json [hunk 2]
```diff
@@ -15,6 +29,44 @@
...
     {"code": "3-0-0", "description": "Food/drink venues", "icon": "fast-food"},
-    {"code": "4-0-0", "description": "Transport", "icon": "car"}
+    {"code": "3-1-0", "description": "Fast food", "icon": "fast-food"},
+    {"code": "3-2-0", "description": "Café", "icon": "fast-food"},
+    {"code": "3-3-0", "description": "Restaurant", "icon": "restaurant"},
+    {"code": "4-0-0", "description": "Transport", "icon": "car"},
+    {"code": "4-1-0", "description": "Railway", "icon": "train"},
+    {"code": "4-1-1", "description": "Train", "icon": "train"},
+    {"code": "4-1-2", "description": "Train station", "icon": "train"},
+    {"code": "4-1-3", "description": "Ticket barrier", "icon": "boom-gate"},
```
Comment (Ashwin-MJ) [inline app/nmwstandard.json]: Are you sure this is a valid icon? I ran it on my device and get a question mark. If you're using icons which aren't Ionicons then I think you'll need to update the code accordingly

File: app/nmwstandard.json [hunk 3]
```diff
@@ -15,6 +29,44 @@
...
+    {"code": "4-1-K", "description": "Platform 14", "icon": "train"},
+    {"code": "4-1-L", "description": "Platform 15", "icon": "train"},
+    {"code": "4-1-M", "description": "Platform 16", "icon": "train"},
+    {"code": "4-1-N", "description": "Platform 17", "icon": "train"},
+    {"code": "4-1-O", "description": "Platform 18", "icon": "train"},
+    {"code": "4-1-P", "description": "Platform 19", "icon": "train"},
+    {"code": "4-1-Q", "description": "Platform 20", "icon": "train"},
+    {"code": "4-2-0", "description": "Bus", "icon": "bus"},
+    {"code": "4-2-1", "description": "Bus", "icon": "bus"},
+    {"code": "4-2-2", "description": "Bus stop", "icon": "bus-stop"},
```
Comment (Ashwin-MJ) [inline app/nmwstandard.json]: Same as above

**Review by Ashwin-MJ — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -30,6 +30,7 @@ jobs:
       - name: Install npm dependencies
         run: |
           cd app
+          npm install
```
Comment (C-Begley) [inline .github/workflows/main.yml:33]: This should be changed to npm install -f

Comment (C-Begley) [reply .github/workflows/main.yml:33]: Actually lets wait til my branch and see what happens

**Review by C-Begley — DISMISSED** [review]

**Review by Andrew-Ritchie — DISMISSED** [review]

**Comment by Ashwin-MJ** [comment]
Have you tested with various beacon names? I can test this in a bit

**Comment by miklasr** [comment]
> Have you tested with various beacon names? I can test this in a bit

I've tested with the two beacons I have but it's quite tedious to change their names so I haven't tested beyond that. I have, however, tried out the different strings for the icon as the value for the location code set on my beacon. That way I at least know the changes I made to the `Icon` have worked.

**Review by Ashwin-MJ — COMMENTED** [review]
Tested with a few different beacon names and works well. However, testing with a screen reader results in "Icon of a font awesome 5 shower" being read out. You'll need to change the accessibilityLabel of the `Icon` component. Happy to approve once that's fixed

**Comment by miklasr** [comment]
> Tested with a few different beacon names and works well. However, testing with a screen reader results in "Icon of a font awesome 5 shower" being read out. You'll need to change the accessibilityLabel of the `Icon` component. Happy to approve once that's fixed

I've changed it to use the current location's description instead of the icon name. That way we won't get any strange output for some icons.

**Review by Ashwin-MJ — APPROVED** [review]

**Pull request merged**